tests/fixtures/composerProvider.yml

```diff
@@ -7,7 +7,7 @@
       },
       "repositories": [
         {"packagist.org": false},
-        { "type": "path", "url": "../repositories/update/*" },
+        { "type": "path", "url": "../repositories/update/*", "canonical": false },
```

Comment:
Composer 2 stops search repositories on the ../repositories/update/ folder. See "Repository Priorities" on the https://php.watch/articles/composer-2

Reply:
Do we need this though? Which test fails if this is left unchanged?

Reply:
```
./vendor/bin/phpunit --testsuite composer2

There was 1 failure:

1) OpenEuropa\ComposerArtifacts\Tests\PluginIntegrationTest::testComposerCommands with data set #0 ('{\n  "name": "openeuropa/main...}\n}\n', 'install', array(array('openeuropa/dependency-1 1.0.0', 'openeuropa/dependency-2 1.0.0'), array('/main/vendor/openeuropa/depen....0.txt', '/main/vendor/openeuropa/depen...r.json', '/main/vendor/openeuropa/depen...r.json'), array('/main/vendor/openeuropa/depen....0.txt')))
Failed asserting that file "/test/composer-artifacts/tests/fixtures/main/vendor/openeuropa/dependency-1/artifact-version-1.0.0.txt" exists.

/test/composer-artifacts/tests/PluginIntegrationTest.php:84
```
Looks like composer 2 finds openeuropa/dependency-1 : 1.1.0 in repositories/update and stops to search it in another repositories (See "Repository Priorities" on the https://php.watch/articles/composer-2). But this version doesn't fit our needs "openeuropa/dependency-1": "1.0.0".